### PR TITLE
create unified BE path for isomorphic visualizations

### PR DESCRIPTION
**Pull request by alxnddr**
### Description

First static visualizations that used javascript for rendering were incomplete duplicates which tried to reproduce the logic of the dynamic visualizations. The logic was spread between Clojure and Javascript. This approach has multiple downsides because all viz features have to be implemented twice and the static chart versions were far behind.

With the new [technical vision](https://www.notion.so/metabase/Viz-Revamp-Research-and-Planning-4198b2726ac64f1d89ad1de6187532d2) for visualizations we will be using the same code both for dynamic and static charts. This PR creates a single BE rendering path for isomorphic visualizations and removes code of the BE code of line/area/bar/combo static charts (FE will be removed further).

### How to verify

1. Create a dashboard with line/area/bar/combo charts and two combined scalars on a single card.
2. Send a test subscription
3. Ensure the placeholders are being rendered

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

**Comment by alxnddr** [comment]
Current dependencies on/for this PR:
* master
  * **PR #34606** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34606" target="_blank">Graphite</a> 
    * **PR #34717** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34717" target="_blank">Graphite</a> 
      * **PR #34772** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34772" target="_blank">Graphite</a> 
        * **PR #34809** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34809" target="_blank">Graphite</a> 
          * **PR #34816** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34816" target="_blank">Graphite</a> 
    * **PR #34703** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34703" target="_blank">Graphite</a> 
      * **PR #34763** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34763" target="_blank">Graphite</a> 
        * **PR #34820** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34820" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metabase/metabase/34820?utm_source=stack-comment).

File: test/metabase/pulse/render/body_test.clj [hunk 1]
```diff
@@ -404,86 +404,6 @@
 (defn has-inline-image? [rendered]
   (some #{:img} (flatten-html-data rendered)))
 
-(defn- render-bar-graph [results]
```
Comment (markbastian) [inline test/metabase/pulse/render/body_test.clj:407]: Is there a reason all of these tests were just removed? Seems like they should be ported over to the new system unless they aren't valid any more.

**Comment by markbastian** [comment]
I noticed that the following form from `dev.render-png` no longer works.

```clojure
  (-> [["A" "B"]
       [1 2]
       [30 20]]
      (render.tu/make-viz-data :line {:goal-line {:graph.goal_label "Target"
                                                  :graph.goal_value 20}})
      :viz-tree
      open-hiccup-as-html)
```

This results in:

```
Execution error (PolyglotException) at <js>/:program (Unnamed:1).
ReferenceError: combo_chart is not defined
```

**Comment by markbastian** [comment]
> removes code of the BE code of line/area/bar/combo static charts (FE will be removed further)

Are we ok with this approach or should we have individual PRs for each chart type where we both remove and replace in a single PR?

**Comment by markbastian** [comment]
This looks like the correct verification.

verify

I'm still concerned that we're wholesale removing something vs. piecemeal replacing.

**Review by EmmadUsmani — APPROVED** [review]

**Pull request merged**